uwblueprint/jump-math
# feat: add routes to assessment editor

## Notion ticket link

[Refactor create-assessment pages to use routes for each page](https://www.notion.so/uwblueprintexecs/Refactor-create-assessment-pages-to-use-routes-for-each-page-0a573129441a478ab75520d4fbbc2947)



## Implementation description
* Long but it works :') This feels like the home stretch somehow

Please see my question below!

Note: reviewing by commit is probably a bad idea. I was unable to make nice commits for this one, sorry 😞 


## Steps to test
1. Play around with creating and editing assessments, and verify you can access all pages without 404 errors. Try reloading the page too!



## What should reviewers focus on?
* 


## Checklist
